../../../bin/face_detection,sha256=wIwTG_7XYDaD0n1ABCO9b1DPdkH3D6k57rOwLBYOHzM,279
../../../bin/face_recognition,sha256=08nUkH4t6gUW522ARORfqxLRYMpqOkMgr22IuthB4IQ,281
face_recognition-1.2.3.dist-info/DESCRIPTION.rst,sha256=SM40q8pfYSr87pbbVLU5YBrhqi2o6g4Uz1s8AQH4cW4,20625
face_recognition-1.2.3.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
face_recognition-1.2.3.dist-info/METADATA,sha256=2Jsp0u1AfwuPr4byw8n8BFw7TL4DdoqZxSpy3DiWauk,21682
face_recognition-1.2.3.dist-info/RECORD,,
face_recognition-1.2.3.dist-info/WHEEL,sha256=o2k-Qa-RMNIJmUdIc7KU6VWR_ErNRbWNlxDIpl7lm34,110
face_recognition-1.2.3.dist-info/entry_points.txt,sha256=WyuxTUy_hPnPbtL2f-cXi65poYLmtHf10QPqDFnzups,139
face_recognition-1.2.3.dist-info/metadata.json,sha256=6wRJ9rNf4wBtDRn_d679os3chi2JqpneyIpI7WUlVbE,1550
face_recognition-1.2.3.dist-info/top_level.txt,sha256=ExXWSKN18HXPKa1CC6ev3GV-Be75gpzSrm4bZElYfbQ,17
face_recognition/__init__.py,sha256=Clf2D3B6krUzQFhqxMtHm6seGR5y4CSiGtYtNjWlBgg,246
face_recognition/__pycache__/__init__.cpython-36.pyc,,
face_recognition/__pycache__/api.cpython-36.pyc,,
face_recognition/__pycache__/face_detection_cli.cpython-36.pyc,,
face_recognition/__pycache__/face_recognition_cli.cpython-36.pyc,,
face_recognition/api.py,sha256=JnxqMRSbyQCnmwyVef66SRmmEpZH7N0SoyfWhOk_jdc,10242
face_recognition/face_detection_cli.py,sha256=jPXOL1_vqWPg3YiZZIiNAqgtPGibSa3hPn3AhpSLYp0,2363
face_recognition/face_recognition_cli.py,sha256=F81oo_6Kl5dT84KWtgVlO2PhRdBZCQ2oV7AJzangISE,4744
